pkg/entity/data_contract/data_contract.go

```diff
@@ -87,26 +79,24 @@ func create(cmd *cobra.Command, args *string) {
 	schemaDefinitionFilename := util.GetStringAndErr(flags, schemaDefinitionFlag)
 	isPublic := util.GetBoolAndErr(flags, publicFlag)
 	contractDefinitionFilename := util.GetStringAndErr(flags, contractDefinitionFlag)
-	contractDefinition := readContractDefinition(&contractDefinitionFilename)
+	dataContract := readContractDefinition(&contractDefinitionFilename)
 
 	projectId := project.GetProjectId(cmd)
 	ref := ref(args)
-	schema := getSchemaDefinition(schemaDefinitionFilename, ref, isPublic)
+	schema := readSchemaDefinition(schemaDefinitionFilename, ref, isPublic)
+
+	// Set the fields which are not part of the contract definition file
+	dataContract.Ref = ref
+	dataContract.IsPublic = isPublic
+	dataContract.ProjectId = projectId
+	dataContract.Schema = schema
+	dataContract.Metadata = &entities.DataContractMetadata{}
 
 	req := &data_contracts.CreateDataContractRequest{
 		ProjectId: projectId,
-		DataContract: &entities.DataContract{
-			KeyField:         contractDefinition.KeyField,
-			IsPublic:         isPublic,
-			ProjectId:        projectId,
-			DataSubjectField: contractDefinition.DataSubjectField,
-			Schema:           schema,
-			Ref:              ref,
-			PiiFields:        contractDefinition.PiiFields,
-			Validations:      contractDefinition.Validations,
-			Metadata:         &entities.DataContractMetadata{},
-		},
+		DataContract: dataContract,
 	}
+	util.ProtoMessageJsonPrettyPrinter{}.Print(req)
```

Comment:
Is this intentional? This should be left to the printer.

Reply:
Oh this was just for debugging purposes, will remove.